src/microbe_stage/particles/Ambientalspatialmaterial.tres

```diff
@@ -0,0 +1,15 @@
+[gd_resource type="SpatialMaterial" load_steps=2 format=2]
```

Comment:
This file should be renamed with PascalCase

Reply:
Hmmm, I tried to rename this file but it doesn't show up on the git changes to add

Reply:
yeah. that's a windows problem. I'll rename it.